homeassistant/components/zwave/__init__.py

```diff
@@ -42,7 +42,11 @@
 from .util import (check_node_schema, check_value_schema, node_name,
                    check_has_unique_id, is_node_parsed)
 
-REQUIREMENTS = ['pydispatcher==2.0.5', 'homeassistant-pyozw==0.1.2']
+REQUIREMENTS = [
+    'pydispatcher==2.0.5', 
+    'homeassistant-pyozw==0.1.2', 
+    'pyserial==3.4'
```

Comment:
All integrations must use the same version.

Also, new or updated dependencies must be added to `requirements_all.txt` by running `script/gen_requirements_all.py`.